### PR TITLE
Fix for JSON keys with a null value in ACF 10+ for ExtraInfo

### PR DESCRIPTION
**Pull request by KrunchMuffin**

### CODE REVIEW

File: client/bugLogService.cfc [hunk 1]
```diff
@@ -558,6 +558,10 @@
 	<cffunction name="sanitizeKeyData" access="private" returntype="struct" hint="removes values from structures for known fields that hold sensitive data">
 		<cfargument name="data" type="struct">
 		<cfset var local = {}>
+		<!--- Fix for JSON keys with a null value in ACF 10+ 8 & 9 would set null values to "null" string --->
+            	<cfif !StructKeyExists(arguments.data,local.i)>
```
Comment (oarevalo) [inline client/bugLogService.cfc]: I think this needs to be inside the `<cfloop>` below, since at this point `local.i` is undefined.

Comment (KrunchMuffin) [inline client/bugLogService.cfc]: oops

**Comment by oarevalo** [comment]
Also, please increment the value of `variables.bugLogClientVersion` at the beginning of `bugLogService.cfc`. This helps to keep track of which version is being used.

**Comment by KrunchMuffin** [comment]
all good?

File: client/bugLogService.cfc [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <cfcomponent>
-	<cfset variables.bugLogClientVersion = "1.8-c5">	<!--- bugloghq client version --->
+	<cfset variables.bugLogClientVersion = "1.8-c7">	<!--- bugloghq client version --->
```
Comment (oarevalo) [inline client/bugLogService.cfc]: what happened to 6?

**Comment by KrunchMuffin** [comment]
I checked what you had and I swear it said 6

**Comment by oarevalo** [comment]
nevermind, you are right; i was looking at the wrong place. 7 is right

**Comment by oarevalo** [comment]
can you solve the merge conflict? Once that is done we can merge the PR.
doesn't look like the conflict is any major thing.

**Comment by oarevalo** [comment]
@KrunchMuffin never mind on the merge conflict. I took care of it.

Thanks for finding and fixing this issue!

**Comment by KrunchMuffin** [comment]
cool. i was at lunch. thanks!

**Pull request merged**